graphql_compiler/cost_estimation/int_value_conversion.py

```diff
@@ -88,8 +88,6 @@ def convert_int_to_field_value(
     See module docstring for details. The int_value is expected to be in the range of
     convert_field_value_to_int.
 
-    Datetimes are returned in the UTC timezone.
-
```

Comment:
The docstring here was already incorrect. So I didn't need to change any of the code.